src/Type/Enum/AvailableLocalesEnum.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WPGraphQL\Type;
+
+if ( ! function_exists( 'wp_get_available_translations' ) ) {
+	require_once ABSPATH . '/wp-admin/includes/translation-install.php';
+}
+
+$values       = [];
+$translations = wp_get_available_translations();
+
+$values[ WPEnumType::get_safe_name( 'en_US' ) ] = [
+	'value'       => 'en_US',
+	'description' => _x( 'English (United States)', 'wp-graphql' ),
+];
```

Comment:
I'm curious why this had to be defined independently? Does this not come through with `wp_get_available_translations`? Or is this meant to be a fallback?